docs/knife_commands.md

```diff
@@ -0,0 +1,92 @@
+Knife commands
+==============
+
+# Modifying a single box
+
+To create a single box, run:
+
+    knife box create <role> -E <env>
+
+You can also optionally add:
+
+* `--index <number>` - This sets the index of node to create. If this is set then the index will be appended to the node name, and the `node_idx` value will be set in `/etc/clearwater/local_config`
+* `--image <name>` - This parameter is used when creating an AMI image, and gives the AMI image the passed in name
+* `--standalone` - This parameter controls whether the box should have the `shared_config` role. This should be set if creating the first box in the environment.
+
+Creating a single box is typically used when creating an [AMI](http://clearwater.readthedocs.org/en/stable/All_in_one_EC2_AMI_Installation/index.html), an [AIO node](http://clearwater.readthedocs.org/en/stable/All_in_one_OVF_Installation/index.html), or spinning up a single box for testing. To change the boxes in an existing deployment you should use the `deployment resize` command described below.
+
+To delete a single box, run:
+
+    knife box delete <box_name - typically <env>-<role>[-<index>]> -E <env>
+
+# Modifying a deployment
+
+To create a deployment, run:
+
+    knife deployment resize -E <env>
+
+You can optionally add:
+
+* `--<box-type>-count` - This controls how many of each box type is created. As the default, chef creates one each of a Bono, Sprout, Homer, Homestead and Ellis.
+* `--apply-shared-config` - If this is set, the shared configuration will be updated on every box after the resize is complete. This is service impacting (as many services need to be restarted to pick up the new configuration), so it isn't run by default. A useful case for this option is when you're resizing your deployment to have Ralf nodes when it didn't previously.
```

Comment:
You say "after the resize is complete" - but you haven't actually talked about resizing a deployment yet, only creating one.